src/Widgets/PopoverWidget.vala

```diff
@@ -73,6 +75,13 @@ public class Power.Widgets.PopoverWidget : Gtk.Grid {
         var show_percent_revealer = new Gtk.Revealer ();
         show_percent_revealer.add (show_percent_box);
 
+        power_mode_list = new PowerModeList (false);
```

Comment:
This argument seems weird. Why are we initializing this class with false?

Reply:
This is place holder code taken from power switchboard plug. This allows usnig a specific settings key based on whether the laptop is on battery or not.

Reply:
Yeah we should make sure to clean up any placeholder stuff before merging please

Reply:
Binding it to dm.has_battery to see if it works.